### PR TITLE
Target size understanding review mg

### PR DESCRIPTION
**Pull request by mbgower**
View [updated Target Size (Minimum) Understanding text](https://raw.githack.com/w3c/wcag/7a8f4eea2f08acdbabb4e89f458381e61280e053/understanding/22/target-size-minimum.html)

### CODE REVIEW

**Comment by detlevhfischer** [comment]
@mbgower I have updated / cleanded up the images and changed the figcaption text of the image with the two rows of buttons - plus a few very minor changes.

**Review by patrickhlauke — APPROVED** [review]

File: understanding/22/target-size-minimum.html
Comment (WilcoFiers) [inline understanding/22/target-size-minimum.html:67]:
```suggestion
        <p>When targets are not 24 by 24 CSS pixels, it is still possible to meet this Success Criteria. The next two illustrations show sets of buttons which are only 20 CSS pixels tall. In the first set, there are no targets immediately above or below the buttons, so they pass. In the second illustration, the buttons have been stacked on top of one another, resulting in a fail.</p>
```

File: understanding/22/target-size-minimum.html
Comment (patrickhlauke) [inline understanding/22/target-size-minimum.html:58]: "in not accidentally triggering adjacent targets" sounds a bit weird. perhaps smoother "not to trigger adjacent targets accidentally"

File: understanding/22/target-size-minimum.html
Comment (patrickhlauke) [inline understanding/22/target-size-minimum.html:61]: "As mobile devices generally have internal heuristics" seems to imply here that the SC is only really aimed at mobile devices, which it isn't. touch on desktop/laptop devices, mouse, stylus, etc are all covered, and this "heuristics on mobile" part doesn't address those scenarios.

File: understanding/22/target-size-minimum.html
Comment (patrickhlauke) [inline understanding/22/target-size-minimum.html:67]: "to meet this Success Criteria" > "to meet this Success Criterion". let's not make that criteria/criterion mess again...

File: understanding/22/target-size-minimum.html
Comment (patrickhlauke) [inline understanding/22/target-size-minimum.html:77]: "the About us menu" > "the <q>About us</q> menu" or similar?

File: understanding/22/target-size-minimum.html
Comment (patrickhlauke) [inline understanding/22/target-size-minimum.html:82]: "the magnifying icon is outside the picture" > "the magnifying icon is outside of the picture" sounds cleaner

**Review by patrickhlauke — CHANGES_REQUESTED** [review]

**Review by detlevhfischer — APPROVED** [review]

File: understanding/22/target-size-minimum.html
Comment (patrickhlauke) [reply understanding/22/target-size-minimum.html:61]: Mulling this over (as it's the last comment that wasn't addressed), perhaps moving that first sentence from where it is now (so starting the section directly with "The following...") to just before the "Spacing versus size" heading, and morphing it to:

```Note: touchscreen devices/user agents generally have internal heuristics to identify which link or control is closest to a user's touch interaction. In these cases, providing sufficient spacing around a target can work as effectively as a larger target size itself.```

would help actually embed this thought more logically into the flow, so it doesn't come out of nowhere AND doesn't seem to just focus the whole section purely on mobile.

**Comment by patrickhlauke** [comment]
See above comment (in case it gets lost in the github UI) https://github.com/w3c/wcag/pull/1770#discussion_r643317319

File: understanding/22/target-size-minimum.html
Comment (alastc) [reply understanding/22/target-size-minimum.html:61]: Got back from holiday, I can't tell whether @patrickhlauke's comments were added before/after the version approved in the meeting, do you know @mbgower?

Comment (patrickhlauke) [reply understanding/22/target-size-minimum.html:61]: after. i do think they're quite important though to remove the weird non-sequitur of talking straight about mobile heuristics, would be good to get a second (third?) approval, otherwise i'll file a new PR with that once merged.

Comment (patrickhlauke) [reply understanding/22/target-size-minimum.html:61]: to be clear, the unresolved one here was after the meeting (as that was the last outstanding one that hadn't been discussed in the meeting, at least going by the minutes)

File: understanding/22/target-size-minimum.html
Comment (patrickhlauke) [inline understanding/22/target-size-minimum.html:61]: perhaps nitpicking but not sure if "the value of spacing" may be misinterpreted by some readers (particularly non-native english speakers) to mean the value/number/magnitude of spacing (i.e. erroneously interpreting this as meaning some touchscreen devices use heuristics to increase spacing).

**Comment by alastc** [comment]
There are plenty of changes here that would be valuable to get published, so let's use the [approval from the meeting](https://www.w3.org/2021/06/01-ag-minutes.html#t06) and a new PR can be created for further updates.

**Pull request merged**
